src/components/Widget/index.js

```diff
@@ -26,24 +26,38 @@ class Widget extends Component {
     event.target.message.value = '';
   }
 
+  getCustomLauncher = () => {
+    if (React.Children.count(this.props.children) > 1) {
+      if (React.Children.map(this.props.children, child => child.type.name).includes('Launcher')) {
+        return (
+          React.createElement(
+            this.props.children[React.Children.map(this.props.children, child => child.type.name).indexOf('Launcher')].type,
+            this.props.children[React.Children.map(this.props.children, child => child.type.name).indexOf('Launcher')].props
```

Comment:
`this.props.children[React.Children.map(this.props.children, child => child.type.name).indexOf('Launcher')]` should be a variable